api/conf/settings/thumbnails.py

```diff
@@ -27,3 +27,7 @@
 THUMBNAIL_HTTP_ERROR_THRESHOLD_TO_NOTIFY = config(
     "THUMBNAIL_HTTP_ERROR_THRESHOLD_TO_NOTIFY", default=1000, cast=int
 )
+
+THUMBNAIL_HTTP_ERROR_THRESHOLD_FOR_REPEATED_NOTIFICATIONS = config(
+    "THUMBNAIL_HTTP_ERROR_THRESHOLD_FOR_REPEATED_NOTIFICATIONS", default=100, cast=int
```

Comment:
Cool! Just maybe this should be higher tho.
```suggestion
    "THUMBNAIL_HTTP_ERROR_THRESHOLD_FOR_REPEATED_NOTIFICATIONS", default=1000, cast=int
```

Reply:
Sure, why not 🙂